test/mail_test.exs

```diff
@@ -482,4 +482,13 @@ defmodule MailTest do
 
     assert result == "Test!"
   end
+
+  test "test plain text parsing newline characters" do
```

Comment:
can you remove the `test "test`. The test should read `test "plain text parsing newline characters"`